pkg/kv/kvserver/mvcc_gc_queue.go

```diff
@@ -64,6 +69,14 @@ const (
 
 	probablyLargeAbortSpanSysCountThreshold = 10000
 	largeAbortSpanBytesThreshold            = 16 * (1 << 20) // 16mb
+
+	// deleteRangePriority is used to indicate replicas that needs to be processed
+	// out of band quickly to enable storage compaction optimization because all
+	// data in them was removed.
+	deleteRangePriority = math.MaxFloat64
+	// gcHintScannerTimeout determines how long hint scanner can hold mvcc gc queue
+	// from progressing
+	gcHintScannerTimeout = 5 * time.Minute
```

Comment:
This value seems reasonable given the 10 minute scanner interval, but let's explain why it was chosen here.